src/braket/circuits/result_type.py

```diff
@@ -197,3 +200,6 @@ def __repr__(self) -> str:
 
     def __copy__(self) -> ObservableResultType:
         return type(self)(observable=self.observable, target=self.target)
+
+    def __hash__(self) -> int:
+        return super().__hash__()
```

Comment:
Would this already inherit the superclass method?

Reply:
__hash__ is not inherited if __eq__ is overwritten. Dropped a link in the code in case anyone has the same question in the future: https://docs.python.org/3/reference/datamodel.html#object.__hash__